cli/cmd/get.go

```diff
@@ -547,16 +547,18 @@ func apiResourceTable(apiGroupStatuses map[string]*resource.APIGroupStatus) stri
 			updatedAt = groupStatus.ActiveStatus.Start
 		}
 
-		if groupStatus.Requested != 0 {
-			rows = append(rows, []interface{}{
-				name,
-				groupStatus.Available(),
-				groupStatus.UpToDate(),
-				groupStatus.Requested,
-				groupStatus.FailedUpdated,
-				libtime.Since(updatedAt),
-			})
+		if groupStatus.Requested == 0 {
```

Comment:
nit: it might be even more clear if we put this check at the beginning of the for loop